src/processing/script/transformers/assignment-destructuring.ts

```diff
@@ -26,20 +26,22 @@ const transformer: Transformer<AssignmentExpression> = {
     condition: node => node.operator === '=' && (node.left.type === Syntax.ObjectPattern || node.left.type === Syntax.ArrayPattern),
 
     run: (node, _parent, _key, tempVars) => {
-        const assignments = [] as (AssignmentExpression | Identifier)[];
-        let firstTemp     = void 0;
+        const assignments = [] as (MemberExpression | AssignmentExpression | Pattern)[];
```

Comment:
Why do we need this change?

Reply:
I tried to link the 'firstTemp' changes to 'assignments' according to TS.